### ISSUE TITLE
no notification on while scrobbling

### ISSUE THREAD

**Issue by y2kbugger** [tgwizard/sls#257]
Expected Result: When  option 'notify now playing' is enabled, I get a notification that scrobbling is working.

Actual result: No notification is shown. under statistics the note playing notification amount seems to be counting them although they are never seen.

running Android M

**Comment by HumbleBeeBumbleBee** [comment]
Try the last.fm app for your desktop computer.
The notifications should show up there.
This may be a change with last.fm website.

**Comment by y2kbugger** [comment]
I'm confused what the website has to do with notifications. I'm talking about notifications on in the phone from the app.

**Comment by omian** [comment]
I'm also having this issue described by @y2kbugger 
My plays are scrobbled to last.fm but I am not receiving notifications. 

It's also unclear to me what it's got to do with the desktop client ( @HumbleBeeBumbleBee )

**Comment by HumbleBeeBumbleBee** [comment]
Where are the notificationsame supposed to be? Ifeed on your phone could you share a link to a screen shot?

**Comment by omian** [comment]
Since they don't show up, there's nothing for me to take a screenshot of.

I'm assuming they're supposed to be a notification in your notification pull down which indicates what track is currently being scrobbled. The official last.fm scrobbler app does this and it is useful because it provides you with easy confirmation that scrobbling is active. Therefore I'm assuming that the 'notify now playing' setting in SLS is supposed to be doing the same thing. But it is only an assumption.

**Comment by HumbleBeeBumbleBee** [comment]
Do you think notifications will be a big deal and people will like them?
I can't add a heart feature in notifications due to backwards compatibility issues.
Although most music apps have this already with track information.

**Comment by omian** [comment]
I don't know how big a deal they are to others. I like them because I like the assurance that my plays are being registered by the scrobbler (as well as the assurance that they're not being registered if I've disabled scrobbling from a particular app). Some people probably won't like having them so I guess it would be good to have an enable/disable toggle.

I wouldn't want multiple notifications when an offline phone goes back online. So yes, I suppose I mean now playing notification.

Basically, I like the behaviour of the official last.fm scrobbler when it comes to notifications.

**Comment by HumbleBeeBumbleBee** [comment]
Hi @omian. The notification is done, but I still need to add an option button in the Options menu.

https://drive.google.com/open?id=0B9lPaMaVVtiJdzF6Tnl2S3RuSFk

**Comment by HumbleBeeBumbleBee** [comment]
Here's another version. Notifications are disabled by default, so people don't rage about it.

https://drive.google.com/open?id=0B9lPaMaVVtiJNXdXNGcwUEUzUnM
